### PR TITLE
Great work, Bob... you're winning this class

### PR DESCRIPTION
**Pull request by rwnilsen3**
please pull me!

### CODE REVIEW

**Comment by asalaices** [comment]
Funny ;) 
:warning:

**Comment by jlahpor** [comment]
what happened?

**Comment by rwnilsen3** [comment]
booya!

File: bios/bob-rules.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+* I am almost 6 feet tall
+* I like tacos
```
Comment (jrussell2) [inline bios/bob-rules.md]: What about burritos?

**Pull request merged**